### PR TITLE
Dash lastodash 

### PR DESCRIPTION
**Pull request by shammamah**
Issue for app: #20 

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/dash-lastodash
- Current gallery app URL: https://dash-gallery.plotly.host/dash-lastodash
- Python app repository link: https://github.com/plotly/lastodash

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-lastodash/runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-3.7.0
```
Comment (ronniebugia) [inline apps/dash-lastodash/runtime.txt]: I think the latest version of Python is 3.7.2

File: apps/dash-lastodash/LICENSE [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+"""
```
[the remark this answers is not in the document]
Comment (shammamah) [reply apps/dash-lastodash/LICENSE]: Yep, you can get rid of this :)

**Comment by ohaanika** [comment]
@christopherjeon Sorry I forgot to tell you earlier, but I think I fixed it and a made a few more tweaks just now. This is how it looks for me:

Screen Shot 2019-08-08 at 05 33 52

Screen Shot 2019-08-08 at 05 34 39

Tell me if it looks any different to you, or if anything feels off?

**Comment by christopherjeon** [comment]
> @christopherjeon Sorry I forgot to tell you earlier, but I think I fixed it and a made a few more tweaks just now. This is how it looks for me:
> 
> Screen Shot 2019-08-08 at 05 33 52
> 
> Screen Shot 2019-08-08 at 05 34 39
> 
> Tell me if it looks any different to you, or if anything feels off?

Just tried it myself and it looks good! Thank you

**Pull request merged**